src/test/java/com/marklogic/kafka/connect/source/ReadRowsViaOpticSerializedTest.java

```diff
@@ -0,0 +1,71 @@
+package com.marklogic.kafka.connect.source;
+
+import com.marklogic.client.document.XMLDocumentManager;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.FileHandle;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ReadRowsViaOpticSerializedTest extends AbstractIntegrationSourceTest {
+
+    @Test
+    void testRowBatcherTask() throws InterruptedException {
+        loadMarkLogicTestData();
```

Comment:
Because this is almost identical to the test for a DSL query, I think the following can be used to avoid the duplication:

1. Make a protected method in the parent class of `verifyQueryReturnsFifteenAuthors` (as that captures explicitly what we expect to get back)
2. Move the helper methods into the parent class as well 

Arguably, you could make a separate class too - e.g. AuthorViewTester or something like that - so that the parent class  doesn't get stuffed with a bunch of helper methods only needed by a couple tests. That's a problem I've run into before with abstract parent test classes. But I think we can cross that bridge later. 

One other rename - `loadMarkLogicTestData` -> `loadFifteenAuthors`, which further makes it explicit what's going on in this scenario (or `loadFifteenAuthorsIntoMarkLogic` in case you think it's ambiguous whether "load" refers to ML or Kafka).